clients/ctl/admin-ui/src/features/dataset/DatasetTable.tsx

```diff
@@ -32,13 +70,18 @@ const DatasetsTable = ({ datasets }: Props) => {
           <Th pl={1}>Name</Th>
           <Th pl={1}>Fides Key</Th>
           <Th pl={1}>Description</Th>
+          <Th pl={1}>Status</Th>
```

Comment:
should we render the status column at all in non-plus?

Reply:
There's still a chance we'll have something meaningful to show for non-classified datasets, but if not this will be easy to update.